docs/en/Specified-agent-config.md

```diff
@@ -0,0 +1,28 @@
+# Locate agent config file by system property
+
+## Supported version
+
+5.0.0-RC+
+
+## What is Locate agent config file by system property ？
+In Default. The agent will try to locate `agent.config`, which should be in the `/config` dictionary of agent package. <br>
+If User set the specified agent config file, The agent will try to load the specified agent config file.<br>
```

Comment:
1. If user **sets** ... config file **through system properties**....
1. ...will try to load file from there.

Don't use br in markdown, especially document.